### PR TITLE
Features/968 up2date master

### PR DESCRIPTION
**Pull request by smpiano**
The document related with this change is https://docs.google.com/document/d/1uT1OSqutFSyAM_909SbMUGnGBPVfNlxlA2CXTRoK_x8/edit?usp=sharing

Related with> https://github.com/GlobalFishingWatch/GFW-Tasks/issues/968

### CODE REVIEW

File: pipe_tools/timestamp.py [hunk 1]
```diff
@@ -139,24 +137,6 @@ def rfc3339strFromTimestamp(ts):
 def timestampFromRfc3339str(s):
     return timestampFromUdatetime(udatetime.from_string(s))
 
-def str2date(s):
```
Comment (enriquetuya) [inline pipe_tools/timestamp.py]: Why did you remove these method? How are you sure it isn't being used in any dataflow process?

File: pipe_tools/timestamp.py [hunk 2]
```diff
@@ -139,24 +137,6 @@ def rfc3339strFromTimestamp(ts):
...
-    """
-    :param s date in string expresed with the format AIRFLOW_DATE
-    :type s string
-    :return datetime representing the string s.
-    """
-    return datetime.strptime(s, AIRFLOW_DATE)
-
-def daterange(start_date, end_date):
```
Comment (enriquetuya) [inline pipe_tools/timestamp.py]: Why did you remove these method? How are you sure it isn't being used in any dataflow process?

File: setup.py [hunk 3]
```diff
@@ -40,13 +32,12 @@
...
     "dill==0.2.6",
     "enum34==1.1.6",
-    "funcsigs==1.0.2",
     "future==0.16.0",
     "futures==3.1.1",
     "gapic-google-cloud-pubsub-v1==0.15.4",
     "google-apitools==0.5.11",
-    "google-auth==1.1.0",
-    "google-auth-httplib2==0.0.2",
+    "google-auth-httplib2==0.0.3",
```
Comment (enriquetuya) [inline setup.py]: I would avoid changing any version in this PR, why did you change them? We  know that current version do work with the dataflow pipe.

File: tests/test_airflow.py [hunk 4]
```diff
@@ -1,193 +0,0 @@
-import pytest
```
Comment (enriquetuya) [inline tests/test_airflow.py]: @smpiano I don't remember, did you  port the test to the new gfw-airflow repo?

File: pickle-json-perf-test.py [hunk 5]
```diff
@@ -9,41 +9,42 @@
 # wrapper class for a dict.  We need a unique class in dataflow to associate with
```
Comment (enriquetuya) [inline pickle-json-perf-test.py]: @smpiano revert the formating of this file.

File: CHANGES.md [hunk 6]
```diff
@@ -1,102 +1,105 @@
...
-* [#50](https://github.com/GlobalFishingWatch/pipe-tools/pull/50)
-  * Pin Airflow version to 1.10.1
-  * NOTE: This changes where not tested on existing pipelines with 1.9.0
+The format is based on [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-0.2.4 - (2018-11-15)
---------------------
+## [Unreleased]
```
Comment (enriquetuya) [inline CHANGES.md]: Add an entry here stating that we are removing all the airflow code and moving to the new repo. Include a link to the new repo.

File: pipe_tools/timestamp.py (hunk 1 above)
Comment (smpiano) [reply pipe_tools/timestamp.py]: @enriquetuya These methods I added to work with BigQueryOperator. They were moved to the `airflow-gfw` repo. The link to the file that it contains the methods is [this](https://github.com/GlobalFishingWatch/airflow-gfw/pull/1/files#diff-f40d9a34dfaaa298eec29db28500f2e6)

File: pipe_tools/timestamp.py (hunk 2 above)
Comment (smpiano) [reply pipe_tools/timestamp.py]: I created only for BigqueryOperator purposes.

File: setup.py (hunk 3 above)
Comment (smpiano) [reply setup.py]: @enriquetuya I will left the versions with them original tags as you suggested. The changes of the tags were referenced in the doc attached but without airflow there is no need to update them.

File: tests/test_airflow.py (hunk 4 above)
Comment (smpiano) [reply tests/test_airflow.py]: @enriquetuya yes, I moved the test where the airflow operators are. I mean, the new `airflow-gfw` repo. Here is the [test_airflow.py](https://github.com/GlobalFishingWatch/airflow-gfw/pull/1/files#diff-a55eeb5db2d79fedfb6b8b772a475204)

File: CHANGES.md (hunk 6 above)
Comment (smpiano) [reply CHANGES.md]: @enriquetuya good idea! Thanks!
